scripts/common.js

```diff
@@ -0,0 +1,20 @@
+import { Client, Intents } from 'discord.js';
+import { config } from 'dotenv';
+import minimist from 'minimist';
+
+export const parseArgs = () => minimist(process.argv.slice(2));

```

Comment:
What is `minimist` used for?

Reply:
Minimist is just a small library that parses the CLI arguments for us. 

```
$ node example/parse.js -x 3 -y 4 -n5 -abc --beep=boop foo bar baz
{ _: [ 'foo', 'bar', 'baz' ],
  x: 3,
  y: 4,
  n: 5,
  a: true,
  b: true,
  c: true,
  beep: 'boop' }
```

https://github.com/substack/minimist

Reply:
I don't think we need a whole library for this. We only have a single flag; the
flag could just be `$1` instead:

```sh
node script.js '{"issue": 30}'
```

```js
const payload = JSON.parse(process.argv[1])
```

Reply:
Good idea :) resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/190/commits/8e78797a78e82e58ddc514f7af0633e04ed748e8>.